src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Color[] colors = Color.values();
```

Comment:
Usually, class-level variables are some services or fields that are part of a bigger entity, like engine is a part of a car, or an email is a detail of some user, but an array like this is more of a part of a method and is not worth to be stored in a class, even if it is a constant value like in this case